oonib/oonibackend.py

```diff
@@ -102,60 +102,71 @@
     multiService.addService(http_return_request_helper)
     http_return_request_helper.startService()
 
-# add the tor collector service here
+def getHSEndpoint(endpoint_config):
+    hsdir = os.path.join(torconfig.DataDirectory, endpoint_config['hsdir'])
```

Comment:
There's also a `tor_datadir` config variable, and I thought it'd be more confusing if one took priority (sometimes). Maybe erroring out is the least confusing?